backend/resolver/aws/aws.go

```diff
@@ -67,6 +69,16 @@ func New(cfg *any.Any, logger *zap.Logger, scope tally.Scope) (resolver.Resolver
 		return nil, errors.New("service was not the correct type")
 	}
 
+	var topologyService topology.Service
+	topologySvc, ok := service.Registry[topology.Name]
+	if ok {
```

Comment:
nit: combine, harder to read this way toplogyService vs toplogySvc

Reply:
I can rename to make it easier to read, but I needed two variables so I can supply `topologyService` to the resolver struct which can be nil. Or i can just rearrange this logic and move the construction above the topology initialization bits and just assign the value eg: `r.topology = topologyService`.

Reply:
yes i meant combine the if to make the scope clearer
```suggestion
	if svc, ok := service.Registry[topology.Name]; ok {
```

Reply:
Oh ok thanks for the clarification I didn't understand at first